app/shared/availability-view/TimelineGroups/TimelineGroup/AvailabilityTimeline/Reservation.js

```diff
@@ -40,13 +40,28 @@ function Reservation({ onClick, ...reservation }) {
       <div>
         <Glyphicon glyph="time" />
         {' '}
-        {startTime.format('HH:mm')} - {endTime.format('HH:mm')}
+        {startTime.format('HH:mm')}
+        {' '}
+-
+        {endTime.format('HH:mm')}
```

Comment:
Using template literals looks much better: 
```suggestion
        {`${startTime.format('HH:mm')} - ${endTime.format('HH:mm')}`}
```